Client/Frontend/Fakespot/Views/FakespotOptInCardView.swift

```diff
@@ -167,26 +167,31 @@ final class FakespotOptInCardView: UIView, ThemeApplicable {
     @objc
     private func didTapLearnMore() {
         viewModel?.onTapLearnMore()
+        viewModel?.dismissViewController?()
     }
 
     @objc
     private func didTapTermsOfUse() {
         viewModel?.onTapTermsOfUse()
+        viewModel?.dismissViewController?()
     }
 
     @objc
     private func didTapPrivacyPolicy() {
         viewModel?.onTapPrivacyPolicy()
+        viewModel?.dismissViewController?()
     }
```

Comment:
I think dismissal of the view controller should be in the viewModel `onTapPrivacyPolicy` call. We want to keep views pretty dumb and comunicate back only that the action happened. Dismissal is too specific and I'm thinking it should actually be part of coordinator layer.

Reply:
Same fot the other ones.

Reply:
good catch, I forgot to delete those calls from the view